src/Menu/Menu.js

```diff
@@ -392,11 +392,11 @@ class Menu extends Component {
       menuValue = menuValue || [];
 
       const itemIndex = menuValue.indexOf(itemValue);
-      const [...newMenuValue] = menuValue;
```

Comment:
That logic [looks](https://babeljs.io/repl/#?babili=false&evaluate=false&lineWrap=false&presets=env%2Creact&targets=&browsers=IE%20%3E%2011&builtIns=false&debug=false&code=const%20%5B...newMenuValue%5D%20%3D%20menuValue%3B%0A) good to me.